test/updateDB.test.js

```diff
@@ -0,0 +1,197 @@
+const path = require('path')
+const fs = require('fs')
+const os = require('os')
+const fsify = require('fsify')({
+  cwd: os.tmpdir(),
+  persistent: false,
+  force: true
+})
+const execSync = require('child_process').execSync
+
+const updateDB = require('../updateDB')
+const BrowserslistError = require('../error')
+
+const root = process.cwd()
+const lastVersion = getLastVersion()
+
+beforeEach(() => {
+  jest.spyOn(console, 'log').mockImplementation(() => true)
+})
+
+afterEach(() => {
+  process.chdir(root)
+})
+
+describe('updateDB function', () => {
+  it('throws an error if the package.json does not exist', async () => {
+    let structure = [
+      {
+        type: fsify.DIRECTORY,
+        name: 'withoutPackageJson',
+        contents: []
+      }
+    ]
+
+    let tree = await fsify(structure)
+    let withoutPackageJson = tree[0].name
+    let message = 'Cannot find package.json'
+
+    process.chdir(withoutPackageJson)
+
+    expect(() => {
+      updateDB()
+    }).toThrow(new BrowserslistError(message))
+  })
+
+  it('throws an error if the lockfile does not exist', async () => {
+    let structure = [
+      {
+        type: fsify.DIRECTORY,
+        name: 'withoutLockfile',
+        contents: [
+          {
+            type: fsify.FILE,
+            name: 'package.json',
+            contents: '{}'
+          }
+        ]
+      }
+    ]
+
+    let tree = await fsify(structure)
+    let withoutLockfile = tree[0].name
+    let message = 'No lockfile found. Run "yarn install" or "npm install"'
+
+    process.chdir(withoutLockfile)
+
+    expect(() => {
+      updateDB()
+    }).toThrow(new BrowserslistError(message))
+  })
+
+  it('prints the latest version of caniuse-lite', async () => {
+    let npmPackage = await createNpmPackage()
+    let message = 'Fetching caniuse-lite ' + lastVersion + '...'
+
+    process.chdir(npmPackage)
+
+    updateDB()
+
+    expect(console.log).toHaveBeenCalledWith(message)
```

Comment:
It will be more readable if you will put a message to the `expect`, rather than using variable